### ISSUE TITLE
HTML Report + JSON Report

### ISSUE THREAD

**Issue by erzz** [arminc/clair-scanner#89]
Great that we get the JSON report for CI purposes - but in the case of failure some poor engineer or repository maintainer needs to pick up the report and interpret it.

Clair's HTML report would be a great addition to the artifacts produced for those people who need to take action a failing scan.

Proposal:
```
--format="html"
--format="json"
--format="all"
```

**Comment by gargworld** [comment]
can clair-scanner produce report in html format?

**Comment by subhamproject** [comment]
yes - i do have same question , I want to create HTML report with clair-scanner is that possible?

**Comment by sfhl** [comment]
Hi there,
it would be great if this was amde possible, at the moment we are using [clairct](https://github.com/jgsqware/clairctl) after running a scan to produce a html report. Thats an additional step, which should not be needed, i guess.

Thanks in advance!
